data/systemd/insights-register.path.in

```diff
@@ -0,0 +1,18 @@
+# This file is part of insights-client.
+#
+# Any changes made to this file will be overwritten during a software update. To
+# override a parameter in this file, create a drop-in file, typically located at
+# /etc/systemd/system/insights-register.path.d/override.conf. Put the desired
+# overrides in that file and reload systemd.
+#
+# For more information about systemd drop-in files, see systemd.unit(5).
+
+[Unit]
+Description=Automatically Register with Red Hat Insights Path Watch
+Documentation=man:insights-client(8)
+
+[Path]
+PathExists=@sysconfdir@/pki/consumer/cert.pem
```

Comment:
Was this meant to be "PathExists" or "PathChanged" ? I forget where we ended up with that.

Reply:
Considering this code will only be enabled when the client is built with `--enable-auto-registration`, in this case, PathExists= is fine to use.